backend/python/migrations/versions/Add_role_and_branch_to_user_model.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: e7f7ce362f62
+Revises: 2d6c26fde419
+Create Date: 2022-06-01 01:25:25.210065
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "e7f7ce362f62"
+down_revision = "2d6c26fde419"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
```

Comment:
it looks like the change to the user didn't get picked up:
```suggestion
     op.execute("ALTER TYPE roles ADD VALUE 'Driver'")
```

Reply:
similarly in downgrade you would need to add code to remove the `Driver`... it may look something like this (taken from https://medium.com/makimo-tech-blog/upgrading-postgresqls-enum-type-with-sqlalchemy-using-alembic-migration-881af1e30abe)
```
op.execute("ALTER TYPE roles RENAME TO roles_old")
op.execute("CREATE TYPE roles AS ENUM('User', 'Admin')")
op.execute((
        "ALTER TABLE users ALTER COLUMN role TYPE roles USING "
        "roles::text::roles"
    ))
op.execute("DROP TYPE roles_old")
```